XenAdmin/MainWindow.cs

```diff
@@ -1448,8 +1448,8 @@ public void UpdateToolbarsCore()
             ShowTab(TabPageDockerDetails, !multi && !SearchMode && isDockerContainerSelected);
 
             bool isPoolOrLiveStandaloneHost = isPoolSelected || (isHostSelected && isHostLive && selectionPool == null);
-            ShowTab(TabPageUSB, !multi && !SearchMode && ((isHostSelected && isHostLive) || isPoolOrLiveStandaloneHost) && !Helpers.FeatureForbidden(selectionConnection, Host.RestrictUsbPassthrough));
             ShowTab(TabPageGPU, !multi && !SearchMode && ((isHostSelected && isHostLive) || isPoolOrLiveStandaloneHost) && Helpers.ClearwaterSp1OrGreater(selectionConnection) && !Helpers.FeatureForbidden(selectionConnection, Host.RestrictGpu));
+            ShowTab(TabPageUSB, !multi && !SearchMode && ((isHostSelected && isHostLive) || isPoolOrLiveStandaloneHost) && !Helpers.FeatureForbidden(selectionConnection, Host.RestrictUsbPassthrough));

```

Comment:
I know that this code hasn't changed in this PR, but I had just noticed that we add the tab for pools as well, which shouldn't be the case. So we should remove `isPoolOrLiveStandaloneHost` from here

Reply:
Indeed, USB is not available for pool. I checked design page.